tmt/steps/discover/fmf.py

```diff
@@ -178,9 +181,9 @@ def options(cls, how=None):
                      'supported. Defaults to the top fmf root if it is '
                      'present, otherwise top directory (shortcut "/").'
                 ),
-            ] + super().options(how)
+            ]) + cast(List[tmt.options.ClickOptionDecoratorType], super().options(how))
```

Comment:
This `cast()` should be unnecessary once #1333 is merged.

Reply:
Yes. But it's not merged yet.

Reply:
Actually we have a workaround for this now